Contents/Code/__init__.py

```diff
@@ -698,8 +694,8 @@ def getImagesFromASS(self, metadata, media, tvdbid, movie, num=0):
     if not entry: Log.Error("tvdbid '%s' is not found in xml file" % tvdbid); return
     for line in filter(None, entry[0].text.strip().replace("\r","\n").split("\n")):
       num += 1; seasonposternum += 1
-      season, posterURL = line.strip().split("|",1); season = str(int(season))
-      posterPath = "seasons/%s-%s-%s" % (tvdbid, season, os.path.basename(posterURL))
+-     season, posterURL = line.strip().split("|",1); season = str(int(season)) #str(int(x)) remove leading 0 from number string
```

Comment:
Last thing pointing out from visual inspection. You have an accidental leading '-'.